### ISSUE TITLE
authSecret deprecated

### ISSUE THREAD

**Issue by meryll** [ziyasal/FireSharp#106]
Hi,
is it still possible to use FireSharp now when AuthSecret needed for connection (FirebaseClient) is deprecated? 

Thanks!

**Comment by noahhusby** [comment]
Yes, it is still possible but not recommended. You can use firebase-authentication-dotnet by step-up-labs for authentication, then the code it returns, you put in the same place as the AuthSecret. Firebase-authentiation-dotnet is made for firebase-database-dotnet, which is an equivilent for FireSharp. I have not had good luck with it, so you can use Firebase-authentication-dotnet and firesharp for authentication. Msg me if you need help
